### ISSUE TITLE
Support for configuring openai proxy 

### ISSUE THREAD

**Issue by sternelee** [Bryley/neoai.nvim#20]
Unable to connect api.openai.com in some countries, so just add config for this

**Comment by Bryley** [comment]
Hey thanks for opening an issue. So you're just looking for a way to change the `api.openai.com` URL within the config to something else?

**Comment by sternelee** [comment]
> Hey thanks for opening an issue. So you're just looking for a way to change the `api.openai.com` URL within the config to something else?

Yes, I think it just like open_api_key_env you did, I use gpt from other resource. Of course, I can also change the file directly
